### ISSUE TITLE
Some enhancement 

### ISSUE THREAD

**Issue by derzhavin3016** [yuphia/Stack#5]
https://github.com/yuphia/Stack/blob/e83aea09c409fac6d1f1da097a9a9ecdb1318e4c/stackFunctions.h#L13
You can initialise it like (you should follow the order of members in structure)
```c
struct dumpInfo info = {
  __LINE__,
  __FILE__,
  __PRETTY_FUNCTION__
};
```
or even (you shouldn’t follow the order)
```c
struct dumpInfo info = {
 .line =  __LINE__,
  .function = __PRETTY_FUNCTION__,
  .file = __FILE__
};
```

**Comment by derzhavin3016** [comment]
It is not what I'm taking about here)

**Comment by yuphia** [comment]
actually it was resolved here))) - 503f49cd43e72ecbee658944449789f8f8e5ceb4
